modules/ROOT/pages/tutorial-setup.adoc

```diff
@@ -1,70 +1,136 @@
 = Prerequisites
 
+== Virtualization with `libvirt`
+
 Those tutorials are written targeting a Linux environment with a working `libvirt` setup and hardware virtualization support via `KVM`. For instructions to setup `libvirt` and `KVM` you may refer to the link:https://docs.fedoraproject.org/en-US/quick-docs/getting-started-with-virtualization/[Getting started with virtualization] guide from Fedora. Although this setup guide is focused on Fedora, those tutorials should work on any distribution with `libvirt` installed.
 
-== Downloading a Fedora CoreOS QCOW2 image
+== CoreOS tools
+
+For those tutorials, we will need the following tools:
+
+  * `coreos-installer`: Tto download the latest Fedora CoreOS QCOW2 image,
+  * `fcct`: To generate Ignition configuration from Fedora CoreOS Config files,
+  * `ignition-validate`: To validate Ignition configuration files.
 
-First of all, we have to download the latest stable release of Fedora CoreOS. As we will use `libvirt`, we will download an archive containing a QEMU image from link:https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[getfedora.org]:
+=== Setup with `podman` or `docker`
+
+All of the tools required to work with Fedora CoreOS are available from containers hosted on link:https://quay.io/[quay.io]:
 
 [source,bash]
 ----
-RELEASE="32.20200715.3.0" # Update this with the latest release from https://getfedora.org/en/coreos
-curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
-curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig
+podman pull quay.io/coreos/coreos-installer:release
+podman pull quay.io/coreos/fcct:release
+podman pull quay.io/coreos/ignition-validate:v2.4.1
 ----
```

Comment:
I think we should just go ahead and define container aliases here and not worry about showing full `podman run` examples.  If we define the aliases then there shouldn't be much difference between the "running in a container" case and the "running via rpm" case, so we could then move the examples later and the examples would just be `coreos-installer download blah blah`, `cat fcct.yaml | fcct > config.ign` etc.

A few variations on the aliases you had are below. Some notable
differences:

- don't use `:Z`, feels safer to not
- add an alias for coreos-installer
- use `$PWD`, just personal preference
- add `-v ${PWD}:/pwd --workdir /pwd` to the `fcct` alias because there is the `fcct -d` option that someone might want to use. 

```
alias coreos-installer='podman run --pull=always --rm -ti \
                        -v ${PWD}:/pwd --workdir /pwd     \
                        quay.io/coreos/coreos-installer:release'
```

```
alias ignition-validate='podman run --pull=always --rm -ti \
                         quay.io/coreos/ignition-validate:release'
```

```
alias fcct='podman run --pull=always --rm -ti \
            -v ${PWD}:/pwd --workdir /pwd     \
            quay.io/coreos/fcct:release'
```

Honestly I could see us converting this whole section into a `coreos tools cheat sheet` outside of the tutorials section at some point.

Reply:
I don't think having pull=always for fcct & ignition-validate make sense as they are short execution time commands and checking for a new release every time is costly from a user perspective. However, You will not probably not run coreos-installer that often so making it pull=always as value.

Reply:
I run `coreos-installer download` quite often (but I'm a developer on the project :smile: )

Reply:
🙂